l-sgov/l-sgov-359-2011-glosář.ttl

```diff
@@ -394,6 +394,19 @@ l-sgov-359-2011-pojem:územní-prvek-nebo-územně-evidenční-jednotka
         <http://www.w3.org/2004/02/skos/core#prefLabel>
                 "územní prvek nebo územně evidenční jednotka"@cs .
 
+l-sgov-359-2011-pojem:uvádí-číslo-orientační-adresy
+        a       <http://www.w3.org/2004/02/skos/core#Concept> ;
+        <http://purl.org/dc/terms/source>
+                <https://esbirka.opendata.cz/zdroj/předpis/359/2011/sekce/6/2/a> ;
+        <http://www.w3.org/2004/02/skos/core#broader>
+                l-sgov-359-2011-pojem:uvádí-adresní-komponent , <https://slovník.gov.cz/základní/pojem/typ-vztahu> ;
+        <http://www.w3.org/2004/02/skos/core#definition>
+                "V prvním řádku se uvede název ulice, číslo popisné nebo evidenční a číslo orientační včetně dodatku9), přitom číslo orientační se uvede za číslo popisné a oddělí se lomítkem."@cs ;
```

Comment:
není "dodatku9)" typo?

Reply:
jo, ten dodatek má odkaz na poznámku pod čaroum to je ta devítka. smažu